YangMingShan/YMSPhotoPicker/Public/YMSPhotoPickerViewController.h

```diff
@@ -37,6 +37,11 @@
  */
 @property (nonatomic, readonly) YMSPhotoPickerTheme *theme;
 
+/**
+ *  @brief Use this property to customize the returned item type for single selection. Default value is NO.
+ */
+@property (nonatomic, assign) BOOL shouldReturnAssetForSingleSelection;
```

Comment:
After discussing with @stewwu, we'd agree return Asset is better behavior.
However, we also want to make this change backward compatible.  Thus, how about renaming this new property to shouldReturnImageForSingleSelection with default YES ?